client/tests/integration/triggers/time_trigger.rs

```diff
@@ -178,6 +181,9 @@ fn mint_nft_for_every_user_every_1_sec() -> Result<()> {
     let (_rt, _peer, mut test_client) = <PeerBuilder>::new().with_port(10_780).start_with_runtime();
     wait_for_genesis_committed(&vec![test_client.clone()], 0);
 
+    // Start listening BEFORE submitting any transaction not to miss any block committed event
+    let event_listener = get_block_committed_event_listener(&test_client)?;
```

Comment:
Shouldn't we create this just before the last `submit()` before submitting sample instructions?

I mean shouldn't we put this before line 240 (in current diff)

Reply:
For what? there is no transactions submitted between this two points and iroha don't produce empty blocks so effect should be the same

Reply:
Yes, indeed